idb/postgres/writer_test.go

```diff
@@ -0,0 +1,1309 @@
+package postgres
```

Comment:
this should move next to`writer.go` or be renamed

Reply:
same